### PR TITLE
Docopt

### PR DESCRIPTION
**Pull request by quantumew**
implement docopt and default filename for hackyanalyze. Did NO refactoring. Figure that will be in a separate pull request.

### CODE REVIEW

File: bin/hackyanalyze [hunk 1]
```diff
@@ -3,12 +3,34 @@ import sys
...
-    output_more = True
+doc = """ hackyanalyze
+
+    Analyzes hackytrack output files. If run without options current day's file
+    will be analyzed.
+
+    Usage: hackyanalyze [options]
+
+    Options:
+        -f --file <FILE>    File to analyze. Defaults to current day tracking file.
```
Comment (kyle-long) [inline bin/hackyanalyze]: Could you make this an optional argument instead?  Works like

```
Usage: hackyanalyze [<filename>] ...
```

**Comment by kyle-long** [comment]
Also could you update the README?  It hints at using docopt in the future but also talks about the "more" argument which will work differently now

File: bin/hackyanalyze (hunk 1 above)
Comment (quantumew) [inline bin/hackyanalyze]: a0cb1eb274700097daa4cf0326a882a7a081c12a - This updates the docs as well

**Comment by kyle-long** [comment]
Looking good

**Pull request merged**